src/popup/components/AppsBrowserListItem.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <div class="apps-browser-list-item-wrapper">
+    <IconWrapper
+      v-if="appIcon"
+      :icon="appIcon"
+      class="app-image"
+      is-boxed
+    />
+    <div
+      v-else
+      class="app-image"
+    />
+    <div class="app-title">
+      {{ appTitle }}
+    </div>
+  </div>
+</template>
+<script lang="ts">
+import { defineComponent } from 'vue';
+import IconWrapper from './IconWrapper.vue';
+
+export default defineComponent({
+  name: 'AppsBrowserListItem',
+  components: { IconWrapper },
+  props: {
+    appTitle: { type: String, required: true },
+    appIcon: { type: Object, default: null },
+  },
+  setup() {
+  },
+});
+</script>
+<style scoped lang="scss">
+@use '../../styles/variables' as *;
+
+.apps-browser-list-item-wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
+  padding-top: 4px;
+
+  &:hover {
+    .app-image {
+      border-radius: 10px;
+      border: 2px solid rgba(255, 255, 255, 0.3);
```

Comment:
1. Use colors from our variables. Here it's `$color-white`.
2. Don't use border for creating outlines as they make the content to decrease the size when the border size changes. Use `box-shadow: inset ...` instead. Take note also that the border size cannot be transitioned smoothly as it transitions by whole pixels. On the opposite box shadow transitions smoothly.